userland/examples/tests/adc/main.c

```diff
@@ -0,0 +1,80 @@
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <tock.h>
+#include <console.h>
+#include <timer.h>
+#include <adc.h>
```

Comment:
prefer to keep headers in alpha order (keep the split, just these 4 need reordering)